### PR TITLE
docs(material/stepper): add harness example for stepper

### PR DESCRIPTION
**Pull request by annieyw**

### CODE REVIEW

File: src/components-examples/material/stepper/stepper-harness/stepper-harness-example.spec.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import {HarnessLoader, parallel} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatStepperModule} from '@angular/material/stepper';
+import {StepperHarnessExample} from './stepper-harness-example';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {ReactiveFormsModule} from '@angular/forms';
+import {STEPPER_GLOBAL_OPTIONS} from '@angular/cdk/stepper';
+
+describe('ToolbarHarnessExample', () => {
```
Comment (crisbeto) [inline src/components-examples/material/stepper/stepper-harness/stepper-harness-example.spec.ts:13]: Should this be renamed?

File: src/components-examples/material/stepper/stepper-harness/stepper-harness-example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatStepperModule, NoopAnimationsModule, ReactiveFormsModule],
+        declarations: [StepperHarnessExample],
+        providers: [{
+          provide: STEPPER_GLOBAL_OPTIONS,
+          useValue: {showError: true} // Required so the error state shows up in tests.
```
Comment (crisbeto) [inline src/components-examples/material/stepper/stepper-harness/stepper-harness-example.spec.ts:28]: I think this part is mostly relevant for our own tests and can be removed from here together with the error state test below.

**Review by andrewseguin — APPROVED** [review]
LGTM

**Comment by mmalerba** [comment]
@annieyw it looks like this is somehow causing the stepper e2e test to time out. My guess is the e2e app page for stepper is just rendering all of the stepper examples, and this new one is probably causing some kind of exception on the page (pure guess with no actual investigation).

**Comment by annieyw** [comment]
fixed e2e test timeout

**Pull request merged**